openage/main/main_cpp.pyx

```diff
@@ -45,9 +41,6 @@ def run_game(args, root_path):
         else:
             args_cpp.mods = vector[string]()
 
-        # create the gil, because now starts the multithread part!
-        PyEval_InitThreads()
```

Comment:
why is this call no longer needed?

Reply:
See https://docs.python.org/3/c-api/init.html#c.PyEval_InitThreads

> Changed in version 3.9: The function now does nothing.